src/utils/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from "./fs-helper";
-export * from "./config-loader";
```

Comment:
As as I set code to give CRLF this is probably as project is relying  on global git config for EOL handling Should be a project setting.